templates/download/iot/intel-iot.html

```diff
@@ -0,0 +1,231 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Download Ubuntu for Intel IoT platforms | Download{% endblock %}
+{% block meta_description %}​​Canonical and Intel have partnered to provide Ubuntu Desktop and Ubuntu Core on Intel's IoT platforms: Atom&reg; X6000E Series, Intel&reg; Pentium&reg; and Celeron&reg; N and J Series and 11th Gen Intel&reg; Core&trade; processors.{% endblock meta_description %}
+{% block meta_image %}https://assets.ubuntu.com/v1/fc79371c-__++_++_+grad+_+en+_+800x418.jpeg{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1>Download Ubuntu for Intel IoT platforms</h1>
+      <p class="p-heading--4">Accelerate industrial compute with Ubuntu on Intel processors</p>
+      <p>The entire family of industrial-grade Ubuntu images optimsed for Intel IoT hardware, from Core to Desktop and Server, are available for download across a wide range of Intel platforms.</p>
+    </div>
+    <div class="col-5 u-hide--medium u-hide--small u-align--center">
+      {{
+        image (
+        url="https://assets.ubuntu.com/v1/efca8490-intel-logo-0.png",
+        alt="",
+        width="300",
+        height="300",
+        hi_def=True,
+        loading="lazy"
```

Comment:
Appears above the "fold"

```suggestion
        loading="auto"
```